### ISSUE TITLE
Ghidra breaks when importing x86-16 target

### ISSUE THREAD

**Issue by Arcnor** [NationalSecurityAgency/ghidra#988]
**Describe the bug**
The provided target (shareware version of an old game) breaks Ghidra when importing with the following errors:

After just selecting the file to import:
(NTHeader$$EnhancerByCGLIB$$5309bf04) Invalid NtHeader c50004

After selecting the language and format (using the defaults that Ghidra has detected):

```
java.lang.IllegalArgumentException: Block must contain split address
	at ghidra.program.database.mem.MemoryMapDB.split(MemoryMapDB.java:785)
	at ghidra.app.util.opinion.MzLoader.adjustSegmentStarts(MzLoader.java:220)
	at ghidra.app.util.opinion.MzLoader.load(MzLoader.java:118)
	at ghidra.app.util.opinion.AbstractLibrarySupportLoader.doLoad(AbstractLibrarySupportLoader.java:346)
	at ghidra.app.util.opinion.AbstractLibrarySupportLoader.loadProgram(AbstractLibrarySupportLoader.java:83)
	at ghidra.app.util.opinion.AbstractProgramLoader.load(AbstractProgramLoader.java:112)
	at ghidra.plugin.importer.ImporterUtilities.importSingleFile(ImporterUtilities.java:401)
	at ghidra.plugin.importer.ImporterDialog.lambda$okCallback$7(ImporterDialog.java:351)
	at ghidra.util.task.TaskLauncher$1.run(TaskLauncher.java:90)
	at ghidra.util.task.Task.monitoredRun(Task.java:126)
	at ghidra.util.task.TaskRunner.lambda$startTaskThread$1(TaskRunner.java:94)
	at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1128)
	at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:628)
	at java.base/java.lang.Thread.run(Thread.java:834)
```
**Attachments**
[target.zip](https://github.com/NationalSecurityAgency/ghidra/files/3591595/target.zip)

**Environment (please complete the following information):**
 - OS: macOS 10.14
 - Java Version: 11.0
 - Ghidra Version: 9.1-DEV (b5498c96)

**Comment by ryanmkurtz** [comment]
Thanks for the report.  Did this binary work in the past or are you testing it for the first time?

**Comment by ryanmkurtz** [comment]
Nevermind, I just tested in 9.0.4 and can see the behavior is different.

**Comment by ryanmkurtz** [comment]
I just tested 9.0.4 and the behavior is a little different (but it still fails to import).

**Comment by Arcnor** [comment]
Thanks for checking. This is the first time I try this binary

**Comment by ryanmkurtz** [comment]
Not sure when exactly this got fixed but it's working now for 9.1.
